ticketevolution/seatmaps-client
# SEAT-70 Created a map legend

## Changelog
- installed `reselect` to handle memoizing derived calculations
- installed `node-sass`, `sass-loader` to compile sass files to css
- installed `@fontawesome/*` to display pretty icons
- installed `classnames` to handle combining classes and conditionally setting classes on components
- installed `normalize-scss` to reset default html styles (only used in `./examples/using-module`)
- created `Legend` component

## Testing Instructions
1. `yarn start`
1. wait for your browser to open the example page
1. you should now see a box in the top right corner of the page
1. there should be text in the box that should read "Show Map Legend"
1. there should be a down arrow to the left of the text
1. click the box
1. the box should have expanded to show multiple options
1. "Show Map Legend" should now read "Hide Map Legend"
1. the down arrow should now be an up arrow
1. there should be a small colored square for each option
1. there should be a cost range for each option
1. the costs should be displayed in rounded dollar amounts
1. none of the costs for the option should overlap
1. click the "Hide Map Legend" box
1. the options should now be hidden
1. "Hide Map Legend" should now read "Show Map Legend
1. the down arrow should now be an up arrow
1. ㊗️ 🎊 🐠 

## Jira Tickets
[SEAT-70](https://ticketevolution.atlassian.net/browse/SEAT-70)
